spec/src/main/asciidoc/core/definition.asciidoc

```diff
@@ -717,6 +736,8 @@ A stereotype may also specify that:
 
 A bean may declare zero, one or multiple stereotypes.
 
+// TODO how about we finally allowed declaring `@Priority` on stereotypes?
```

Comment:
What is the use case? You actually want to avoid having (for instance) multiple interceptors that all have the same priority because then you don't know which one goes first. And in case of alternatives, it is probably even an error to have two with identical priority.

Reply:
https://issues.redhat.com/browse/CDI-695 :-)